pulldown-cmark/src/scanners.rs

```diff
@@ -269,6 +269,26 @@ impl<'a> LineStart<'a> {
         }
     }
 
+    pub(crate) fn scan_spoiler_fence(&mut self) -> bool {
+        if self.scan_case_insensitive(b":::") {
+            if self.scan_case_insensitive(b" spoiler ") {
```

Comment:
I think it's a mistake to implement this in a way that `:::spoiler` doesn't work.